playbooks/solrcloud.yml

```diff
@@ -4,6 +4,8 @@
   become: true
   vars_files:
     - ../site_vars.yml
+    - ../group_vars/solrcloud.yml
```

Comment:
I'm not sure why I needed to include this here, but the playbook didn't work without it

Reply:
That's weird — the only things in that file that I see are DataDog-related.

Reply:
Oh, I was confused between production and staging.  Still weird that you have to include the file, but it's not mixed up between production and staging at least.